README.md

```diff
@@ -1,55 +1,129 @@
-    - `UpTree` propagate the tags from descendants to parents up the hierarchy
-    ![image](https://user-images.githubusercontent.com/7566324/62931481-ba630c00-bdbe-11e9-9cdf-6d05e955721b.png)
-    
-    
-    - `DownTree` inherites the tags by descendants from their parrent down the hierarchy
-    ![image](https://user-images.githubusercontent.com/7566324/62931421-a3241e80-bdbe-11e9-8f02-fd22d0fbcc6f.png)
+The main concept of VC Personalization module is to personalize the issuing of catalog, price lists and marketing actions using Tags and User Groups.
 
-### Settings:
-Manage the tags propagation policies:
-![image](https://user-images.githubusercontent.com/7566324/62931587-eaaaaa80-bdbe-11e9-8693-e1694f104f22.png)
+The Contacts should be first included into  specific User Groups and then assign  catalogs, price lists or promotions to this UG.
 
-### Example:
+## Key features
 
-<details><summary>Managing catalog tags</summary><p>
-  
-![catalog-tags](https://user-images.githubusercontent.com/7536694/31497152-6510aac0-af5e-11e7-9e99-8f3f2d45cca3.gif) </p></details>
+1. Control the visibility of catalog objects such as Product and Categories, through manual tagging of these objects with special property `Tags` which can contain multiple predefined values which also may be defined for customer profile
+1. Inheritance `Tags` between catalogs objects depend on catalog taxonomy (categories inherit this property from parent catalog or category, products inherit this property from parent category)
+1. Allow using `User groups` from customer profile to products or categories filtration in the storefront search
+1. Different policies for `Tags` propagation in catalog objects hierarchy:
+    1. `UpTree` propagate the tags from descendants to parents up the hierarchy
+    ![image](https://user-images.githubusercontent.com/7566324/62931481-ba630c00-bdbe-11e9-9cdf-6d05e955721b.png);
+    1. `DownTree` inherits the tags by descendants from their parent down the hierarchy
+    ![image](https://user-images.githubusercontent.com/7566324/62931421-a3241e80-bdbe-11e9-8f02-fd22d0fbcc6f.png).
 
-<details><summary>Managing category and product tags</summary><p>
+## Scenarios
 
-![category-product-tags](https://user-images.githubusercontent.com/7536694/31497224-b5e4b1a8-af5e-11e7-90a1-918f120efddd.gif) </p></details>
+### Create New User Group
 
-#### Important! Don't forget to rebuild search indexes for categories and products after tagging process.
+1. Go to More->Contacts module-> select a user from the list and open user's details;
+1. Click the 'Edit' of the User Grpups;
+1. In the opened 'Member groups dictionary' enter the new User Group name and click the 'Add' button;
+1. Save the new user group;
+1. The new user group is now available for selection.
+
+![Add user group](docs/media/screen-add-usergroup.png
+)
+
+![User group added](docs/media/screen-usergroup-added.png)
+
+![User group available](docs/media/screen-usergroup-available.png)
```

Comment:
Huge pictures, can you make them in 100% scale?

Reply:
Place pictures in the list under the corresponding item

Reply:
done